Python/tracemalloc.c

```diff
@@ -1258,6 +1260,10 @@ _PyTraceMalloc_TraceRef(PyObject *op, PyRefTracerEvent event,
                         void* Py_UNUSED(ignore))
 {
     if (event != PyRefTracer_CREATE) {
+        /* we don't want bother here with the lock for performance reasons */
+        if (_Py_atomic_load_int32_relaxed(&tracemalloc_config.tracing)) {
```

Comment:
`tracemalloc_config.tracing` is an `int` (as opposed to `int32_t`)

```suggestion
        if (_Py_atomic_load_int_relaxed(&tracemalloc_config.tracing)) {
```

Reply:
I swear I looked for that at some point and thought it didn't exist... (`_Py_atomic_load_int_relaxed`)